### PR TITLE
New airflow version 2.6.3, 2.7.2

### PR DESCRIPTION
**Pull request by maltesander**
# Description

- added support for 2.6.3, 2.7.2
- removed support for 2.2.3, 2.2.4, 2.2.5, 2.4.1
- adapted python imports to be compatible with 2.7.2 (`airflow.www.fab_security.manager` -> `flask_appbuilder.const`)

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [x] Changes are OpenShift compatible
- [ ] CRD changes approved
- [x] Helm chart can be installed and deployed operator works
- [x] Integration tests passed (for non trivial changes)
- [x] Changes need to be "offline" compatible
```

```[tasklist]
# Reviewer
- [x] Code contains useful comments
- [x] (Integration-)Test cases added
- [ ] Documentation added or updated
- [x] Changelog updated
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

### CODE REVIEW

File: CHANGELOG.md
Comment (siegfriedweber) [inline CHANGELOG.md:1]:
```diff
- - `vector` `0.26.0` -> `0.31.0` ([#308]).
+ - `vector` `0.26.0` -> `0.33.0` ([#308], [#334]).
```

**Review by adwk67 — APPROVED** [review]
I noticed we are getting deprecation messages in e.g. the worker logs that apparently come from config settings that exist in their old location e.g. [core] (old) instead of [database] (new) - however, all our configs seem to be in the right place so I'm not sure where that is coming from. LGTM!

**Comment by maltesander** [comment]
test https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/airflow-operator-it-custom/104/

File: docs/modules/airflow/partials/supported-versions.adoc [hunk 1]
```diff
@@ -2,8 +2,6 @@
 // This is a separate file, since it is used by both the direct Airflow-Operator documentation, and the overarching
 // Stackable Platform documentation.
 
-- 2.2.3
-- 2.2.4
-- 2.2.5
-- 2.4.1
-- 2.6.1
+- 2.7.2
```
[the remark this answers is not in the document]
Comment (adwk67) [reply docs/modules/airflow/partials/supported-versions.adoc:5]: Thanks!

**Comment by maltesander** [comment]
another try https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/airflow-operator-it-custom/107/

**Comment by maltesander** [comment]
last one hopefully, 1 worker more memory:
https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/airflow-operator-it-custom/108/

**Review by adwk67 — APPROVED** [review]
LGTM!

**Pull request merged**